raw/src/requests/send_audio.rs

```diff
@@ -1,51 +1,60 @@
 use std::borrow::Cow;
-use std::ops::Not;
 
 use crate::requests::*;
 use crate::types::*;
 
 /// Use this method to send an audio
-#[derive(Debug, Clone, PartialEq, PartialOrd, Serialize)]
+#[derive(Debug, Clone, PartialEq, PartialOrd)]
 #[must_use = "requests do nothing unless sent"]
-pub struct SendAudio<'s, 'c, 'p, 't> {
+pub struct SendAudio<'c, 'p, 't> {
     chat_id: ChatRef,
-    audio: Cow<'s, str>,
-    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
Why remove this? Does this happen by default?

Reply:
Optional fields are handled by `multipart_map!` now.